tests/std/tests/VSO_0180469_ptr_cat/test.compile.pass.cpp

```diff
@@ -502,3 +502,46 @@ void test_Lex_compare_optimize() {
     test_case_Lex_compare_optimize_pr<less>();
     test_case_Lex_compare_optimize_pr<greater>();
 }
+
+#ifdef __cpp_lib_concepts
+// Also test GH-1523, in which std::equal does not properly convert non-pointer contiguous iterators to pointers.

```

Comment:
I find "does not" to be slightly confusing. The comment would make more sense if it described the bad behavior in past tense.